### DISCUSSION TITLE
[Feature Request] Sort Image by Random

### DISCUSSION THREAD

**Discussion by djnixy** [zanllp/infinite-image-browsing#462] [Ideas]
Hi,

Add feature to sort image randomly please?

**Comment by zanllp** [comment]
Could you please explain the significance of adding the feature to sort images randomly?

**Reply by djnixy** [reply]
Could help on getting new ideas/inspiration by looking back to what we have.

**Reply by ZYFsir** [reply]
How about scroll down randomly
